### ISSUE TITLE
the optimizer's learning rate

### ISSUE THREAD

**Issue by yangfengwxy** [tmulc18/Distributed-TensorFlow-Guide#15]
Hello, why can't the optimizer's learning rate use placeholders in distributed code?
